### PR TITLE
User Groups table function to twig 

### PR DESCRIPTION
**Pull request by tabraiz961**
Signed-off-by: Tabraiz Malik <tabraizmalik96@gmail.com>

### Description

Please describe your pull request.

Fixes #

Before submitting pull request, please review the following checklist:

- [ ] Make sure you have read our [CONTRIBUTING.md](https://github.com/phpmyadmin/phpmyadmin/blob/master/CONTRIBUTING.md) document.
- [ ] Make sure you are making a pull request against the correct branch. For example, for bug fixes in a released version use the corresponding QA branch and for new features use the _master_ branch. If you have a doubt, you can ask as a comment in the bug report or on the mailing list.
- [ ] Every commit has proper `Signed-off-by` line as described in our [DCO](https://github.com/phpmyadmin/phpmyadmin/blob/master/DCO). This ensures that the work you're submitting is your own creation.
- [ ] Every commit has a descriptive commit message.
- [ ] Every commit is needed on its own, if you have just minor fixes to previous commits, you can squash them.
- [ ] Any new functionality is covered by tests.

### CODE REVIEW

**Comment by tabraiz961** [comment]
hi @williamdes UnitTest with Random order gives 
Failed asserting that '' contains "index.php?route=/server/status/processes"
can you help me out with a hint here ?

**Comment by tabraiz961** [comment]
Hi @williamdes  i'd like to continue some work in the same file(User Groups) for twig files. Should i continue or wait for your review|commit ?

**Comment by williamdes** [comment]
> can you help me out with a hint here ?

They are broken and some work needs to be done to fix them, nothing related to this PR ;)

File: libraries/classes/Server/UserGroups.php [hunk 1]
```diff
@@ -74,96 +75,82 @@ public static function getHtmlForListingUsersofAGroup($userGroup)
...
-            $html_output .= '</tbody>';
-            $html_output .= '</table>';
-            $html_output .= '</form>';
         }
+        $addUserUrl = Url::getFromRoute('/server/user-groups', ['addUserGroup' => 1]);
+        $addUserIcon = Generator::getIcon('b_usradd');
         $GLOBALS['dbi']->freeResult($result);
+        $template = new Template();
+        return  $template->render('server/user_groups/user_groups', [
+            'user_group_msg' => __('User groups'),
```
Comment (williamdes) [inline libraries/classes/Server/UserGroups.php]: You could use the twig `trans` filter in the template
You can find examples in other twig templated

**Review by williamdes — APPROVED** [review]

**Comment by williamdes** [comment]
I thing you can continue, if the PR becomes to big you can split your work on other PRs to make review easier

File: libraries/classes/Server/UserGroups.php (hunk 1 above)
Comment (tabraiz961) [reply libraries/classes/Server/UserGroups.php]: oh great i,ll fix it right away. 

> You could use the twig `trans` filter in the template
> You can find examples in other twig templated

Oh great, i'll fix it right away.

**Comment by williamdes** [comment]
@tabraiz961 I personally use GitKraken and would recommend you to use it

**Comment by williamdes** [comment]
@tabraiz961 what this PR a duplicate of #15796 ?

**Comment by tabraiz961** [comment]
> @tabraiz961 what this PR a duplicate of #15796 ?

sorry its just Git is very difficult for me. Give me a moment to answer your question.

**Comment by tabraiz961** [comment]
> @tabraiz961 what this PR a duplicate of #15796 ?

Yes they are same, since my previous commit was in pending so i did create a branch worked on it and pushed it.

**Comment by williamdes** [comment]
Okay, I will reset your master branch so you can have a fresh new start, please never use master or QA_5_0 branches for pull-requests
Instead always create a branch from them so you can do actions without impacting future and previous PRs ;)

**Pull request closed without merging**